apm-server.yml

```
@@ -381,6 +342,14 @@ output.elasticsearch:
   # Configure http request timeout before failing an request to Elasticsearch.
   #timeout: 90
 
+  # The bulk request size threshold, in bytes, before flushing to {es}.
+  # The value must have a suffix, e.g. `"1MB"`. The default is `5MB`.
+  #flush_bytes: 5MB
```

The default is 1MB per bulk indexer, not 5. https://github.com/elastic/go-docappender/blob/44c0e9a0c3f6b6a9b03db5903487feab8a489617/config.go#L56-L60
```suggestion
  # The value must have a suffix, e.g. `"2MB"`. The default is `1MB`.
  #flush_bytes: 1MB
```

---

Maybe we can update that doc and make sure it is up to date

---

done: https://github.com/elastic/apm-server/pull/10377/files#diff-516d6962ac5880d959e69af3915eda1550f31aeebb7bd6e5aafb4059287eb4b4L437